singularity-eos/eos/eos_base.hpp

```diff
@@ -261,6 +261,9 @@ class EosBase {
     return;
   }
 
+  PORTABLE_INLINE_FUNCTION
+  Real RhoPmin(const Real temp) const { return 0.0; }
```

Comment:
Does adding a function to the base class mean that our using statements need to change as well for CRTP?

Reply:
I'd think so, but doesn't the PTE test use analytic EOS, so wouldn't an issue be exposed there?

Reply:
The PTE test uses analytic EOS that should require this, so I'd think it would throw an error if there was an issue

Reply:
oh interesting, yeah the CRTP on cppreference doesn't have the usings so I guess not.